Sources/MongoSwift/ClientSession.swift

```diff
@@ -216,3 +173,57 @@ public final class SyncClientSession {
         }
     }
 }
+
+/**
+ * A MongoDB client session for use with synchronous clients, databases, and sessions.
+ * This class represents a logical session used for ordering sequential operations.
+ *
+ * To create a client session, use `startSession` or `withSession` on a `SyncMongoClient`.
+ *
+ * If `causalConsistency` is not set to `false` when starting a session, read and write operations that use the session
+ * will be provided causal consistency guarantees depending on the read and write concerns used. Using "majority"
+ * read and write preferences will provide the full set of guarantees. See
```

Comment:
this diff has made me notice an existing typo here: this should be "concerns" not "preferences"

Reply:
fixed.